modules/gitops-argo-cd-installation.adoc

```diff
@@ -17,9 +17,9 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 . Select *OpenShift GitOps Operator* from the installed operators and select the *Argo CD* tab.
 
-. Click *Create* to configure the parameters:
+. Click *Create ArgoCD* to configure the parameters:
 
-.. Enter the *Name* of the instance. By default, the *Name* is set to *argocd*. 
+.. Enter the *Name* of the instance. By default, the *Name* is set to *example*. 
```

Comment:
```suggestion
.. Enter the *Name* of the instance. By default, the *Name* is set to `example`. 
```
For reference: https://redhat-documentation.github.io/supplementary-style-guide/#text-entry
Also here: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting

Reply:
Fixed. PTAL.

Reply:
LGTM, thank you!